### ISSUE TITLE
[librdkafka] How to rebuild it

### ISSUE THREAD

**Issue by vipcxj** [microsoft/vcpkg#39039]
### Is your feature request related to a problem? Please describe.

To build librdkafka with sasl support, I need to install libsasl2-dev in the host machine before vcpkg building it. I didn't realize it until my program ran and threw an exception. Because no port feature named sasl. Since the port is built, I need to rebuild it without any changes that VCPKG can perceive.Because of the cache, I can't do this.I tried `vcpkg --help`, and I didn't see the relevant commands. I also found that there was a lot of cache under ~/.cache/vcpkg. But the composition of these cache is not what humans can understand. So I have to delete the entire folder, which causes all my PORT to be rebuilt

### Proposed solution

add a command to rebuild special port, or just remove the cache of it

**Comment by dg0yt** [comment]
Doesn't vcpkg print which cached artifact file it uses? You could delete just that file.

- System dependencies are froned upon. They aren't tracked for caching, and they can pull in unwanted transitive dependencies. Ports are expected to actively avoid them in most cases.
- Caching is controlled by ABI hashes. And the hash is the artifact filename.
  https://learn.microsoft.com/en-us/vcpkg/reference/binarycaching#abi-hash
- You can control how the cache is used when installing.
  - [`--binarysource=<source>`](https://learn.microsoft.com/en-us/vcpkg/commands/common-options#binarysource)
  - `--no-binarycaching`
- `--editable` disables caching, too.

**Comment by vipcxj** [comment]
No, vcpkg print a dir relate to cache, I ls to it and found it just the port dir which include portfile.cmake, not binary. I delete it, nothing happen.

> * You can control how the cache is used when installing.
>   
>   * [`--binarysource=<source>`](https://learn.microsoft.com/en-us/vcpkg/commands/common-options#binarysource)
>   * `--no-binarycaching`
> * `--editable` disables caching, too.

I need cache, I just don't need the cache relate to special port.
